types/three/three-typedarrayutils.d.ts

```diff
@@ -0,0 +1,23 @@
+// https://github.com/mrdoob/three.js/blob/master/examples/js/TypedArrayUtils.js
+
+export namespace TypedArrayUtils {
```

Comment:
Tests would be great to show how using this in TS would look.

Reply:
The doc comments in the original have a decent minimal example.